### DISCUSSION TITLE
Send `POST` requets

### DISCUSSION THREAD

**Discussion by albertcito** [DamianOsipiuk/vue-query#49] [Q&A] [answered]
I would like to use the same code to do `POST` requests. Could I use `useQuery` in some way that it's not saved in the cache? I would like to use the same variables  `isLoading`, `isError`, `isFetching`, `data`, `error`, `refetch` but not auto-caching it.

**Comment by wobsoriano** [comment, marked as answer]
Set `cacheTime` to 0.

But why not use `useMutation` tho?

**Reply by albertcito** [reply]
Lmao,  I didn't know that `useMutation` existed. Thank you very much.
